modules/widgets/social-icons.php

```diff
@@ -388,277 +416,256 @@ public function get_svg_icon( $args = array() ) {
-				'url'   => 'amazon.ca',
-				'icon'  => 'amazon',
-				'label' => 'Amazon',
-			),
-			array(
-				'url'   => 'amazon.com',
+				'url'   => array(
+					'amazon.cn',
+					'amazon.in',
+					'amazon.fr',
+					'amazon.de',
+					'amazon.it',
+					'amazon.nl',
+					'amazon.es',
+					'amazon.co',
+					'amazon.ca',
+					'amazon.com',
+				),
 				'icon'  => 'amazon',
 				'label' => 'Amazon',
 			),
 			array(
-				'url'   => 'apple.com',
+				'url'   => array( 'apple.com' ),
 				'icon'  => 'apple',
 				'label' => 'Apple',
 			),
 			array(
-				'url'   => 'itunes.com',
+				'url'   => array( 'itunes.com' ),
 				'icon'  => 'apple',
 				'label' => 'iTunes',
 			),
 			array(
-				'url'   => 'bandcamp.com',
+				'url'   => array( 'bandcamp.com' ),
 				'icon'  => 'bandcamp',
 				'label' => 'Bandcamp',
 			),
 			array(
-				'url'   => 'behance.net',
+				'url'   => array( 'behance.net' ),
 				'icon'  => 'behance',
 				'label' => 'Behance',
 			),
 			array(
-				'url'   => 'codepen.io',
+				'url'   => array( 'codepen.io' ),
 				'icon'  => 'codepen',
 				'label' => 'CodePen',
 			),
 			array(
-				'url'   => 'deviantart.com',
+				'url'   => array( 'deviantart.com' ),
 				'icon'  => 'deviantart',
 				'label' => 'DeviantArt',
 			),
 			array(
-				'url'   => 'digg.com',
+				'url'   => array( 'digg.com' ),
 				'icon'  => 'digg',
 				'label' => 'Digg',
 			),
 			array(
-				'url'   => 'discord.gg',
-				'icon'  => 'discord',
-				'label' => 'Discord',
-			),
-			array(
-				'url'   => 'discordapp.com',
+				'url'   => array( 'discord.gg', 'discordapp.com' ),
 				'icon'  => 'discord',
 				'label' => 'Discord',
 			),
 			array(
-				'url'   => 'dribbble.com',
+				'url'   => array( 'dribbble.com' ),
 				'icon'  => 'dribbble',
 				'label' => 'Dribbble',
 			),
 			array(
-				'url'   => 'dropbox.com',
+				'url'   => array( 'dropbox.com' ),
 				'icon'  => 'dropbox',
 				'label' => 'Dropbox',
 			),
 			array(
-				'url'   => 'etsy.com',
+				'url'   => array( 'etsy.com' ),
 				'icon'  => 'etsy',
 				'label' => 'Etsy',
 			),
 			array(
-				'url'   => 'facebook.com',
+				'url'   => array( 'facebook.com' ),
 				'icon'  => 'facebook',
 				'label' => 'Facebook',
 			),
 			array(
-				'url'   => '/feed/',
+				'url'   => array(
+					'/feed/',        // WordPress default feed url.
+					'/feeds/',       // Blogspot and others.
+					'/blog/feed',    // No trailing space WordPress feed, could use /feed but may match unexpectedly.
+					'format=RSS',    // Squarespace and others.
+					'/rss',          // Tumblr.
+					'/.rss',         // Reddit.
+					'/rss.xml',      // Moveable Type, Typepad.
+					'http://rss',
+					'https://rss',
+					'rss=1',
+					'/feed=rss',     // Catches feed=rss / feed=rss2.
+					'?feed=rss',     // WordPress non-permalink - Catches feed=rss / feed=rss2.
+					'?feed=rdf',     // WordPress non-permalink.
+					'?feed=atom',    // WordPress non-permalink.
+					'http://feeds',  // FeedBurner.
```

Comment:
Would `http://feeds.` (note the dot) be better here?

Reply:
I think that would be a safe change to make, it is primarily intended to catch Feedburner feed urls so they would all be `http://feeds.`

Reply:
done in b82da9ec4ddb0ca3198fa52be6fbdae6ae207999